tests/devices/test_default_qubit_jax.py

```diff
@@ -89,6 +89,26 @@ def circuit(x):
         # Just test that it works and spits our the right value.
         assert jnp.isclose(circuit(p), expected, atol=tol, rtol=0)
 
+    def test_qubit_circuit_different_interface(self, tol):
+        """Test that the device provides the correct

```

Comment:
```suggestion
        """Test that the interface provides the correct
```
Was this what you meant?